ob_intern/Controller/Short Video List/ShortVideoListViewModel.swift

```diff
@@ -34,6 +37,19 @@ class ShortVideoListViewModel {
     }
    
     // MARK: - Functions
+    func getVideoList() {
+        ShortVDOService().getShortVDOList { [weak self] videoList in
+            guard let self = self else { return}
+            currentList = videoList
+            guard let delegate = delegate else { return }
+            delegate.updateData()
+        } errorHandler: { error in
+            let alert = UIAlertController(title: "Error", message: "Oops, something went wrong. Please try again later.", preferredStyle: UIAlertController.Style.alert)
```

Comment:
อันนี้มันใช้รูปย่อได้ด้วยนะ
```suggestion
            let alert = UIAlertController(title: "Error", message: "Oops, something went wrong. Please try again later.", preferredStyle: .alert)
```